products/pwa.md

```diff
@@ -1,6 +1,6 @@
 # PWA
 
-If you think about providing a decoupled storefront for customers you can use **Shopware PWA** as a base to do that. It is a Single Page Application based on JavaScript and Vue.js, that already integrates a whole lot of the Shopware functionality you know into a very extensible frontend application.
+If you think about providing a decoupled Storefront for customers, you can use **Shopware PWA** as a base. It is a Single Page Application (SPA) based on JavaScript and Vue.js that already integrates a whole lot of the Shopware functionality you know into a very extensible frontend application.
```

Comment:
```suggestion
If you think about providing a decoupled storefront for customers, you can use **Shopware PWA** as a base. It is a Single Page Application (SPA) based on JavaScript and Vue.js that already integrates a whole lot of the Shopware functionality you know into a very extensible frontend application.
```

Lowercase "storefront" is fine here, as it is not referring to the our component "Storefront", but the general concept of a storefront.